infinitecsolutions/infally
# Basic Adaption of project to Infinitec

This PR contains a little clean-up and initial rule changes to be more in-line with the changes we have made to the API guidelines. 

Due to unfortunate design of the tool, rules reside in their respective class (e.g. "CamelCaseForQueryParamsRule.kt') AND a central 'rules-config.conf' file. Both need to be changed for a modified rule.

I am not a Java/Gradle/... expert, but the project seems a little taped together. Happy to receive everybody's thoughts on that.

### How to Verify

1. Check out this branch
2. Run './build-and-run.sh' (first build after first check out might take some time)
3. Go to http://localhost:8080
4. Take an OpenAPI specification and paste it into the editor. The editor cannot resolve included spec snippets, it needs a singular file.
5. Hit Validate
6. See if only violations are reported that are in line with [our guidelines](https://internal-docs.nova.infinitec.solutions/api-guidelines/)
